config/initializers/statsd.rb

```diff
@@ -26,21 +26,42 @@ class << Samson
 end
 
 ActiveSupport::Notifications.subscribe("job_queue.samson") do |*, payload|
+  # deploys running/queued
+  deploys_metrics = payload.delete(:deploys)
+  deploys_metrics.each { |key, value| Samson.statsd.gauge "deploys.#{key}", value }
+
+  # total jobs queued/running reporting
   payload.each { |key, value| Samson.statsd.gauge "job.#{key}", value }
 end
 
+ActiveSupport::Notifications.subscribe("job_status.samson") do |*, payload|
+  Samson.statsd.increment "#{payload[:type]}.#{payload[:status]}"
```

Comment:
have a command prefix like `jobs.` so it's easy to figure out what belongs into it

Reply:
prefer .fetch since it's useless if anything is missing